spec/unit/provider/package/cab_spec.rb

```diff
@@ -18,7 +18,7 @@
 
 require "spec_helper"
 
-describe Chef::Provider::Package::Cab do
+describe Chef::Provider::Package::Cab, :windows_only do
```

Comment:
unit tests shouldn't need `:windows_only` -- if you need this it indicates you're probably missing some kind of a stub/mock